tools/c7n_azure/c7n_azure/utils.py

```diff
@@ -530,3 +535,40 @@ def get_keyvault_secret(user_identity_id, keyvault_secret_id):
 
     kv_client = KeyVaultClient(credentials)
     return kv_client.get_secret(secret_id.vault, secret_id.name, secret_id.version).value
+
+
+@lru_cache()
+def get_service_tag_list():
+    """ Gets service tags, note that the region passed to the API
+    doesn't seem to do anything, so we use a fixed one to improve caching"""
+
+    from c7n_azure.session import Session
+    s = local_session(Session)  # type: Session
+
+    client = s.client('azure.mgmt.network._network_management_client.NetworkManagementClient')
+
+    return client.service_tags.list('westus')
```

Comment:
We should file a bug for this, because I can see a new version of this SDK will result in breaking this query,

Reply:
https://docs.microsoft.com/en-us/rest/api/virtualnetwork/servicetags/list 

Yeah I confirmed all of them return the same value :)  Weird.  I submitted a docs bug.